### ISSUE TITLE
The implementation of Train

### ISSUE THREAD

**Issue by WenB777** [IDEA-Research/HumanSD#8]
Hello authors, your work is quite good, I learned a lot, can you share The implementation of HumanSD with heatmap-guided loss, thank you for your help

**Comment by juxuan27** [comment]
Hi, @WenB777 ! Thank you for your focus.
We plan to public the training code after the paper is received.
